README.md

```diff
@@ -333,26 +337,6 @@ token.EVM.ETH.USDC.address
 crypto.USDC.address
 token.EVM.ETH.address
 ```
-
-`getAddress(domain, 'ETH', 'USDC')` will lookup records in the following order:
```

Comment:
🤔 why this is removed?

Reply:
to be clear, this method is not supported by ENS naming service but for UNS we still do